custom_components/zaptec/api.py

```diff
@@ -29,174 +38,150 @@
 
 # to Support running this as a script.
 if __name__ == "__main__":
+    from const import (API_RETRIES, API_URL, CONST_URL, FALSY, MISSING,
+                       TOKEN_URL, TRUTHY)
+    from misc import mc_nbfx_decoder, to_under
+    from validate import validate
+
     # remove me later
     logging.basicConfig(level=logging.DEBUG)
-    TOKEN_URL = "https://api.zaptec.com/oauth/token"
-    API_URL = "https://api.zaptec.com/api/"
-    CONST_URL = "https://api.zaptec.com/api/constants"
-
-    def to_under(word) -> str:
-        """helper to convert TurnOnThisButton to turn_on_this_button."""
-        # Ripped from inflection
-        word = re.sub(r"([A-Z]+)([A-Z][a-z])", r"\1_\2", word)
-        word = re.sub(r"([a-z\d])([A-Z])", r"\1_\2", word)
-        word = word.replace("-", "_")
-        return word.lower()
+    logging.getLogger("azure").setLevel(logging.WARNING)
 
 else:
-    from .const import API_URL, CONST_URL, TOKEN_URL
-    from .misc import to_under
+    from .const import (API_RETRIES, API_URL, CONST_URL, FALSY, MISSING,
+                        TOKEN_URL, TRUTHY)
+    from .misc import mc_nbfx_decoder, to_under
+    from .validate import validate
 
 
-class AuthorizationFailedException(Exception):
-    pass
+class ZaptecApiError(Exception):
+    '''Base exception for all Zaptec API errors'''
 
 
-# should be a static method of account
-async def _update_remaps() -> None:
-    wanted = ["Observations"]
-    obs = {}
-    async with aiohttp.request("GET", CONST_URL) as resp:
-        if resp.status == 200:
-            data = await resp.json()
-            for k, v in data.items():
-                if k in wanted:
-                    obs.update(v)
-                    # Add names.
-                    obs.update({value: key for key, value in v.items()})
+class AuthorizationError(ZaptecApiError):
+    '''Authenatication failed'''
```

Comment:
Run black and fixup so ' -> "

Reply:
I haven't run black on anything. I didn't know if you did. So I'll run everything through black and update the PR.